docs/concepts.md

```diff
@@ -62,6 +62,24 @@ Each record is uniquely identified by a `record key` and mapped to a file id for
 and file id, never changes once the first version of a record has been written to a file. In short, the
  `file id` identifies a group of files, that contain all versions of a group of records.
 
+## Terminlogies
+
+ * `Hudi Dataset` 
+    A structured hive/spark table whose ingestion and querying is managed by Hudi. Hudi supports both partitioned and 
+    non-partitioned Hive tables. 
+ * `Commit` 
+    A commit refers to one round of ingestion that brings in a batch of new (inserts) and updated records to a dataset.
+    Hudi maintains  monotonically increasing timestamps to track commits and guarantees that a commit is atomically 
+    published.
+ * `Commit Timeline`
+    Commit Timeline refers to the sequence of Commits that was applied in order on a dataset over its lifetime. 
+ * `File Slice` 
```

Comment:
FileSlice is a little more than just the version of a file id, but also includes any delta files and their versions, need a way to incorporate this in your text